atm.js

```diff
@@ -1,3 +1,121 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+  var checkingInput = $("#checking .input").val()
+  var savingsInput = $("#savings .input").val()
+  var checkingTotal = 0
+  var savingsTotal = 0
+  var newCheckingTotal = 0
+
+  function checkingColor(){
+    if (checkingTotal == 0){
+      $("#checking .balance").addClass("zero");
+    }
+    else{
+      $("#checking .balance").removeClass("zero");
+    }
+  }
+  function savingsColor(){
+    if (savingsTotal == 0){
+      $("#savings .balance").addClass("zero");
+    }
+    else{
+      $("#savings .balance").removeClass("zero");
+    }
+  }
```

Comment:
Notice how `savingsColor` and `checkingColor` are almost exactly the same? How about:

```js
function balanceColor(account){
    if (checkingTotal == 0){
        $("#" + account + " .balance").addClass("zero");
    }
    else{
        $("#" + account + " .balance").removeClass("zero");
    }
}
```